### ISSUE TITLE
Validation of renamed feature

### ISSUE THREAD

**Issue by Stormur** [UniversalDependencies/tools#93]
I am in the process of changing some denominations of relations in the three active Latin treebanks, specifically those with the `cmpr` subtype, which are now [standardised to `cmp`](https://github.com/UniversalDependencies/docs/issues/874).

Now, documentation should be OK,  but I do not understand why [the "official" validator](http://quest.ms.mff.cuni.cz/udvalidator/cgi-bin/unidep/validation-report.pl) is still signalling `advcl:cmp` as an error for the ITTB, while everything works fine for UDante and LLCT. Is it just a matter of time?

**Comment by dan-zeman** [comment]
It works for me but I had to trigger [on-line revalidation](http://quest.ms.mff.cuni.cz/udvalidator/cgi-bin/unidep/validation-report.pl?UD_Latin-ITTB) manually.

ITTB was the first Latin treebank to revalidate after the documentation was changed, and it almost looks like its validation started before the local tools repository on the server got updated. Normally this would not happen but this time the validation of ITTB seems to have been invoked quickly several times in a row, before the previous run finished. Renaming the files in documentation may have been the first trigger, then probably ten seconds later registering the feature in the web interface was the second trigger. Such a situation results in several processes writing to the same validation report, which is obviously a problem. There is no mechanism to prevent this from happening, but if a new revalidation is triggered after all these processes finished, it should again provide the correct picture of the data.
